advanced/Templates/pihole-FTL-poststop.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env sh
+
+# Source utils.sh for getFTLPIDFile(), getFTLAPIPortFile()
+PI_HOLE_SCRIPT_DIR='/opt/pihole'
+utilsfile="${PI_HOLE_SCRIPT_DIR}/utils.sh"
+. "${utilsfile}"
```

Comment:
I think you will need to add `# shellcheck disable=SC2154`
or use the same method used here:
https://github.com/pi-hole/pi-hole/blob/fe8b0e98ab64ae47a6c31eea67cede3cf502d831/pihole#L12-L24

Reply:
Ah, using `readonly` satisfies shellcheck? How does it find the actual script location, since on a Pi-hole host it has a different path (not only absolute) than on the (checked out) repository.

But `readonly` is a good idea anyway.

Reply:
Okay, not sure how Stickler CI calls shellcheck, sadly none of the directives to load the actual script works:
```sh
# shellcheck source=../Scripts/utils.sh
# shellcheck source=SCRIPTDIR/../Scripts/utils.sh
# shellcheck source-path=SCRIPTDIR/../Scripts
```
😢. So it's:
```sh
# shellcheck disable=SC1090
```